wordpress-mobile/WordPress-Android
# Redirect to username/pass login on email_login_not_allowed

Fixes #7922

This PR enables a redirect to the username/password screen when the `email_login_not_allowed` error is detected from WordPress.com

FluxC PR: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1211

I will update this PR when the FluxC PR gets merged to point to the merged hash.

To test:

No steps available.
If you are determined though, you might want to rig your WordPress.com sandbox to short-circuit the `WPCOM_Log_In_Via_Email::suspicious_email_login()` function to always fail and point the app to your sandbox. Then, try to login a valid WordPress.com email, choose the "password instead" flow on the magic link screen and try to login in the email/password screen. App should redirect to the username/password screen with a Toast.

Update release notes:
